tests/integration/test_crunching.py

```diff
@@ -1,40 +1,48 @@
 from os import walk
-from os.path import abspath, dirname, join, isfile
-import subprocess
+from os.path import join, isfile, isdir
 
 
+from click.testing import CliRunner
 import pandas as pd
 import numpy as np
 from openscm.scmdataframe import ScmDataFrame
 
+import netcdf_scm
+from netcdf_scm.cli import crunch_data
+
 
 from conftest import TEST_DATA_KNMI_DIR, TEST_DATA_MARBLE_CMIP5_DIR
 
 
 def test_crunching(tmpdir):
-    here = abspath(dirname(__file__))
-
-    THRESHOLD_PERCENTAGE_DIFF = 10 ** -1
-
-    SCRIPT_TO_RUN = join(here, "..", "..", "scripts/crunch_to_scm.py")
     INPUT_DIR = TEST_DATA_MARBLE_CMIP5_DIR
     OUTPUT_DIR = str(tmpdir)
-    VAR_TO_CRUNCH = "tas"
-    command = [
-        "python",
-        SCRIPT_TO_RUN,
-        INPUT_DIR,
-        OUTPUT_DIR,
-        "--var-to-crunch",
-        VAR_TO_CRUNCH,
-        "-f",
-    ]
-    subprocess.check_call(command)
+    VAR_TO_CRUNCH = ".*tas.*"
+
+    runner = CliRunner()
```

Comment:
Nice! I always struggled to test argparse cli's

Reply:
Click is so nice